numbat/modules/core/booleans.nbt

```diff
@@ -0,0 +1,2 @@
+fn and(a: bool, b: bool) = if a then b else false
+fn or(a: bool, b: bool) = if a then true else b
```

Comment:
Is it worth to add these? Or should we just add logical operators?

Reply:
`and` and `or` are so fundamental to booleans, that I would expect that either these functions or infix operators would be provided out-of-the-box

Reply:
I agree: https://github.com/sharkdp/numbat/issues/196